Automattic/jetpack
# Admin bar: update to match with Nav Redesign i3

Related to:

- https://github.com/Automattic/dotcom-forge/issues/8144

## Proposed changes:

This PR updates the admin bar for WordPress.com sites as follows:

1. Repurposes the WordPress logo as a link to /sites.
2. Adds Reader menu.
3. Hides the updates icon (via CSS).
4. Hides the comments icon (via CSS).

---

I also took the opportunity to split the `wpcom-site-menu` feature (which is getting too large) into 3:

- `wpcom-admin-bar`
- `wpcom-admin-menu`
- `wpcom-sidebar-notice`

With this, the responsibility of each feature is clear and localized.

|wp-admin|
|-|
|Before<br><img width="354" alt="image" src="https://github.com/Automattic/jetpack/assets/1525580/500f2f52-29d0-46f4-a0a6-ccab0f28f1ed">|
|After<br><img width="346" alt="image" src="https://github.com/Automattic/jetpack/assets/1525580/83059785-b4e3-41fb-8bf7-b1a61a39433b">|

|site front-end|
|-|
|Before<br><img width="535" alt="image" src="https://github.com/Automattic/jetpack/assets/1525580/fef4fcda-556c-472a-a7f2-5c9a32481e9c">|
|After<br><img width="522" alt="image" src="https://github.com/Automattic/jetpack/assets/1525580/94ad8ec4-26b5-4ac4-8753-0651b87b7e02">|

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Patch this to WordPress.com Features on Jetpack Beta Tester.
3. Smoke-test wp-admin and site frontend and verify the screenshot.
4. Also test on Simple/Atomic Classic/Default sites (4 combinations).
5. (Regression testing) Test that you can still view the sidebar upsell in all cases.
